protocol/contracts/amo/test/RamosTestnetAuraStaking.sol

```diff
@@ -53,6 +53,10 @@ contract RamosTestnetAuraStaking is IAuraStaking, TempleElevatedAccess {
         revert("unimplemented");
     }
 
+    function setRewardTokens(address[] memory /*_rewardTokens*/) external override pure {
+        revert("unimplemented");
```

Comment:
nit: I think a couple of abstract contracts could benefit if this is a global `error Unimplemented();`

Reply:
Good call - done